reinforcement_learning/testconfig.yml

```diff
@@ -0,0 +1,11 @@
+train_instance_count:
+    - 1
+    - 2
+
```

Comment:
This will generate 8 combination of the integ tests, are all of them running in parallel?

Reply:
Also, some of the integ tests here are identical. e.g for local_mode == True, instance type will be ignored so that make some of them are running the same tests

Reply:
None of them are parallel. Thats the improvement that needs to be done.

Reply:
Yeah, I agree, there will be a repetition of tests. This needs to be refactored again.